locales/en.yml

```diff
@@ -277,6 +277,8 @@ en:
     newest_first: "Newest First"
     navigation: Navigation
     next_comment: "Go to the next comment"
+    toggle_search: "Open search"
```

Comment:
Could you add them in spanish too?

```
toggle_search: "Abrir búsqueda"
next_queue: "Próxima cola"
```

Reply:
Sure! French and Portuguese are going to be translated by someone else? Or should I attempt to translate?